docs/Actions/load-user.md

```diff
@@ -0,0 +1,164 @@
+---
+id: LoadUser
+title: Load User
+sidebar_label: Load User
+---
+
+An action that allows altering the user which the subsequent actions refer to. The user change affects all actions after the `Load User` action and after all actions have finished executing the user will be reverted back to the initial one (i.e. the logged-in user). By default, all actions that need a user for their execution will refer to the currently logged in user.
+
+This action also supports loading multiple users by providing multiple user identifiers. The last user identifier provided will also be used as the context user under which all actions execute. Loading multiple users creates a hidden context entity that is leveraged by the following actions which support this:
+
+- [Grant Role](/docs/Actions/grant-role.md)
+- [Revoke Role](/docs/Actions/revoke-role.md)
+- [Authorize User](/docs/Actions/authorize-user.md)
+- [Unauthorize User](/docs/Actions/unauthorize-user.md)
+- [Delete User](/docs/Actions/delete-user.md)
+- [Send Email](/doc/Actions/send-email.md).
+
+When used in conjunction with actions such as [Grant Role](/docs/Actions/grant-role.md), the `Grant Role` action will grant the role specified to all users loaded by this action.
+
```

Comment:
```suggestion

Calling the `Load User` action the second time inside the same workflow will replace the user or the list of users in the context. It never appends to the list that is already loaded.

```

Reply:
This is not the actual behavior, I committed this change with the proper explanation.